### ISSUE TITLE
Which name for the project, for the tool itself ?

### ISSUE THREAD

**Issue by Grenadingue** [hello-world-org/hello-world-emerge#6]
We talked about "hello world" and kind of agreed about it, because it was the best idea we had. For now. It symbolize the fact, for a human, to be able to "say hello" to his/our world, to connect and organize himself/his group(s) with other people while using the project. That's why this formulation is visible in the organization and in this repository name. Do we keep it ? Feel free to suggest ANYthing else.

**Comment by tomacf** [comment]
Huge topic that scares me,  I'm not able to answer without losing 4days of reflexion, I don't know what do you think about this but maybe it would be efficient to ask opinion and proposition from people around us, or people skilled in this kind of problem

**Comment by Grenadingue** [comment]
Yeah huge topic. I think that those 4 days of reflection, if they're needed, will no be loosed in any way. But I agree, we should ask to people around us.

**Comment by Grenadingue** [comment]
As the project is about organizing people between them, about making them able to decide how they will take actions together, and then finally about taking this actions. I thought about **organizaction**, or the french version **organisaction**.
An other idea, that highlights the fact that people will be theoretically able to fully choose how they will organize together, that highlights the extremely modular organization processes we want to make available, I found **modulorg**.
